dist/panzoom.js

```diff
@@ -115,6 +115,7 @@ function createPanZoom(domElement, options) {
     zoomTo: publicZoomTo,
     zoomAbs: zoomAbs,
     smoothZoom: smoothZoom,
+    smoothZoomToAbs: smoothZoomToAbs,
```

Comment:
Can we change this to `smoothZoomAbs()` so that it is symmetrical to `zoomAbs()`?

Reply:
Sure, I've committed an update.